src/DIRAC/Core/Tornado/scripts/tornado_start_CS.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+########################################################################
+# File :   tornado-start-CS
+# Author : Louis MARTIN
+########################################################################
+# Just run this script to start Tornado and CS service
+# Use dirac.cfg (or other cfg given in the command line) to change port
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__RCSID__ = "$Id$"
+
+import os
+import sys
+from DIRAC.Core.Utilities.DIRACScript import DIRACScript
+
+
+@DIRACScript()
+def main():
+  # Must be defined BEFORE any dirac import
+  os.environ['DIRAC_USE_TORNADO_IOLOOP'] = "True"
```

Comment:
what's wrong with keeping it at the top ?

Reply:
I thought I had commented asking you if 8a104f9 was okay but it seems to have been lost, maybe I left it as a draft comment by accident.

My question was: Can this be set after importing `DIRACScript` or will that break things that aren't caught by the integration tests? Now that these are being imported as Python modules I don't like the side effect of "if you import this file weird things will start happening". I'm fairly sure they won't be imported, but I also don't want to waste a few hours only to realise that they were.

Reply:
Looking at what `DIRACScript` contains, I expect it should still work if we put it after. I understand your argument about magic stuff happening when importing. At the same time, this is exactly what you want in this case... and there should be no reason whatsoever to import that if not to execute it.

Reply:
I could imagine doing things like importing all entrypoints to perform some kind of inspection of them. I don't actually have any plans to do this but if it ever becomes useful I doubt we'll remember this caveat. 

If it's that tricky to get right I think a better place for this variable would be in the runit "`run`" scripts themselves and then `tornado-start-CS` can exit with an error if the variable isn't set.

Reply:
I doubt this will ever have a side effect but fine. I think it's fine if we set it where you placed it.